lib/chai/core/assertions.js

```diff
@@ -215,6 +215,16 @@ module.exports = function (chai, _) {
   Assertion.addChainableMethod('contains', include, includeChainingBehavior);
   Assertion.addChainableMethod('includes', include, includeChainingBehavior);
 
+  // TODO: Add jsdoc
+
+  function assertFail (msg) {
+    if (msg) flag(this, 'message', msg);
+    var obj = flag(this, 'object');
+    throw new chai.AssertionError(msg, {}, obj);
+  }
```

Comment:
I agree with the consistency. I chose this route because I was trying to be consistent with the interface. Namely, that values are usually passed along the chain with `expect`. I'll try reworking it.